templates/Billing/budgets-transfer-costs.yaml

```diff
@@ -0,0 +1,104 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Parameters:
+
+  budgetName:
+    Type: String
+
+  emailAddress:
+    Type: String
+    Default: ''
+
+  budgetAmount:
+    Type: Number
+
+Conditions:
+
+  hasSubscription: !Not [ !Equals [ '', !Ref emailAddress ] ]
+
+Resources:
+
+  NotificationTopic:
+    Type: AWS::SNS::Topic
+    Properties:
+      DisplayName: Budgets Notification
+      TopicName: !Sub '${budgetName}-topic'
+
+  EmailSubscription:
+    Type: AWS::SNS::Subscription
+    Condition: hasSubscription
+    Properties:
+      Endpoint: !Ref emailAddress
+      Protocol: email
+      TopicArn: !Ref NotificationTopic
+
+  NotificationTopicPolicy:
+    Type: AWS::SNS::TopicPolicy
+    Properties:
+      Topics:
+        - !Ref NotificationTopic
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: AllowBudgetsPublish
+            Effect: Allow
+            Principal:
+              Service: budgets.amazonaws.com
+            Action: sns:Publish
+            Resource: !Ref NotificationTopic
+
+  Budget:
+    Type: AWS::Budgets::Budget
+    Properties:
+      Budget:
+        BudgetLimit:
+          Amount: !Ref budgetAmount
+          Unit: USD
+        TimeUnit: MONTHLY
+        BudgetType: COST
+        CostTypes:
+          IncludeSupport: false
+        CostFilters:
+          UsageType:
+            # Found using Cost Explorer
+            - "DataTransfer-Out-Bytes"
+            - "AP-DataTransfer-Out-Bytes"
+            - "AU-DataTransfer-Out-Bytes"
+            - "CA-DataTransfer-Out-Bytes"
+            - "EU-DataTransfer-Out-Bytes"
+            - "JP-DataTransfer-Out-Bytes"
+            - "US-DataTransfer-Out-Bytes"
+            - "USE1-EU-AWS-Out-Bytes"
+            - "USE1-APE1-AWS-Out-Bytes"
+            - "USE1-APN1-AWS-Out-Bytes"
+            - "USE1-APS1-AWS-Out-Bytes"
+            - "USE1-APS3-AWS-Out-Bytes"
+            - "USE1-EUW2-AWS-Out-Bytes"
+            - "USE1-USE2-AWS-Out-Bytes"
+            - "USE1-USW1-AWS-Out-Bytes"
+            - "USE1-USW2-AWS-Out-Bytes"
+      NotificationsWithSubscribers:
+        - Notification:
```

Comment:
seems spam induced with all these notifications?

Reply:
To be honest I didn't think too much about them when I made the copy of `templates/Billing/budgets.yaml`, they seemed reasonable at first glance. I see you wrote the original template a few years ago, has the number of notifications for the current budgets been an issue?

Reply:
The only place where I see `budgets.yaml` being used is to setup notifications for general account budgets.   TBH, the only time where I've seen alerts are from accounts for scientific projects which I ignore because i have not idea whether it's warranted.  Although I think they are already kinda spammy.  I guess i don't i don't mind piling but just wanted to point it out.

Reply:
ah ok, thanks for pointing it out. I think we can start here and adjust when we need to